tools/core2scala.rkt

```diff
@@ -131,7 +131,7 @@
       (format "~a: Real" (fix-name (if (list? var) (car var) var)))))
   (with-output-to-string
     (λ ()
-      (printf "\tdef ~a(~a): Real = {\n" (fix-name name) (string-join arg-strings ", "))
+      (printf "\tdef ~a(~a): Real = {\n" (if (dict-has-key? properties ':name) (format "`~a`" (dict-ref properties ':name)) (format "ex~a" index)) (string-join arg-strings ", "))
```

Comment:
Do we not need to escape backticks in the name?

Reply:
That is correct. Is there an easy way to do this? I know neither Scheme nor Racket.

Reply:
Inside the `format` call, you want to replace `(dict-ref properties ':name)` with

    (string-replace (dict-ref properties ':name) "`" "\\`")

Reply:
And, I suppose, another layer of `string-replace` to replace backslashes with two backslashes.

Reply:
Turns out you can't escape backticks, so they're now being replaced by simple apostrophes (if anyone really needs to use them).